tests/foreman/cli/test_myaccount.py

```diff
@@ -72,63 +94,69 @@ def test_positive_update_language(self):
         @Status: Manual
         """
 
-    @stubbed()
+    @tier1
     def test_positive_update_password(self):
-        """@Test: Update Password/Verify fields in My Account
+        """Update Password in My Account
 
         @Feature: My Account - Positive Update
 
-        @Steps:
-        1. Update Password/Verify fields with all variations in [1]
-
         @Assert: User is updated
-
-        @Status: Manual
         """
-
-    @stubbed()
+        password = gen_string('alphanumeric')
+        User.update({
+            'id': self.user['id'],
+            'password': password,
+        })
+        User.foreman_admin_password = password
+        result = User.info({'id': self.user['id']})
+        self.assertTrue(result)
+
+    @tier1
     def test_negative_update_first_name(self):
-        """@Test: Update My Account with invalid FirstName
+        """Update My Account with invalid FirstName
 
         @Feature: My Account - Negative Update
 
-        @Steps:
-        1. Update Current user with all variations of FirstName in [2]
-
         @Assert: User is not updated. Appropriate error shown.
-
-        @Status: Manual
         """
+        with self.assertRaises(CLIReturnCodeError):
+            User.update({
+                'id': self.user['id'],
+                'firstname': gen_string('alphanumeric', 300),
```

Comment:
Alphanumeric only? Why not `invalid_names_list()`? I understand a chance of failure is really low here, but still i'd prefer using our common approach (also it will be easier to update all the negative boundary tests in one place in case we decide to do so)